component/src/test/java/org/wso2/extension/siddhi/store/cassandra/ContainsInCassandraTableTestCase.java

```diff
@@ -0,0 +1,276 @@
+        log.info("containsCassandraTableTest1");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        String streams = "" +
+                "define stream StockStream (symbol string, price float, volume long); " +
+                "define stream CheckStockStream (symbol string, volume long); " +
+                "@Store(type=\"cassandra\", column.family=\"" + TABLE_NAME + "\", " +
+                "keyspace=\"" + KEY_SPACE + "\", client.port=\"" + getPort() + "\", " +
+                "username=\"" + USER_NAME + "\", " +
+                "password=\"" + PASSWORD + "\", " +
+                "cassandra.host=\"" + getHostIp() + "\")" +
+                "@PrimaryKey(\"symbol\")" +
+                "define table StockTable (symbol string, price float, volume long); ";
+        String query = "" +
+                "@info(name = 'query1') " +
+                "from StockStream " +
+                "insert into StockTable ;" +
+                "" +
+                "@info(name = 'query2') " +
+                "from CheckStockStream[(StockTable.symbol==symbol) in StockTable] " +
+                "insert into OutStream;";
+
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams + query);
+        siddhiAppRuntime.addCallback("query2", new QueryCallback() {
+            @Override
+            public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
+                EventPrinter.print(timeStamp, inEvents, removeEvents);
+                if (inEvents != null) {
+                    for (Event event : inEvents) {
+                        inEventCount++;
+                        switch (inEventCount) {
+                            case 1:
+                                Assert.assertEquals(event.getData(), new Object[]{"IBM", 100L});
+                                break;
+                            case 2:
+                                Assert.assertEquals(event.getData(), new Object[]{"WSO2", 100L});
+                                break;
+                            default:
+                                Assert.assertEquals(2, inEventCount);
+                        }
+                    }
+                    eventArrived = true;
+                }
+                if (removeEvents != null) {
+                    removeEventCount = removeEventCount + removeEvents.length;
+                }
+                eventArrived = true;
+            }
+        });
+
+        InputHandler stockStream = siddhiAppRuntime.getInputHandler("StockStream");
+        InputHandler checkStockStream = siddhiAppRuntime.getInputHandler("CheckStockStream");
+        siddhiAppRuntime.start();
+
+        stockStream.send(new Object[]{"WSO2", 55.6f, 100L});
+        stockStream.send(new Object[]{"IBM", 55.6f, 100L});
+        checkStockStream.send(new Object[]{"IBM", 100L});
+        checkStockStream.send(new Object[]{"WSO2", 100L});
+        Thread.sleep(1000);
```

Comment:
We should ideally not be using thread sleeps. Use  SiddhiTestHelper.waitForEvents() instead